### PR TITLE
WID-164 Add Literal enum-like

### PR DESCRIPTION
**Pull request by rarescodemart**

### CODE REVIEW

File: xai_components/xai_storage/store_results.py [hunk 1]
```diff
@@ -32,14 +33,14 @@ class StoreResultsToDrive(ComponentWithWidget):
...
-    H5_format: InArg[bool]
+    format: InArg[Literal['h5', 'npz']]
 
     def __init__(self):
         self.done = False
         self.is_hpc_launch = False
         self.collab_name = InArg('')
         self.folder_path = InArg('')
-        self.H5_format = InArg(False)
+        self.format = InArg('h5')
```
Comment (liadomide) [inline xai_components/xai_storage/store_results.py:43]: To maintain the same behaviour as before I think we should have `npz` here

File: xai_components/xai_storage/store_results.py [hunk 2]
```diff
@@ -116,14 +117,14 @@ class StoreResultsToBucket(ComponentWithWidget):
...
-    H5_format: InArg[bool]
+    format: InArg[Literal['h5', 'npz']]
 
     def __init__(self):
         self.done = False
         self.is_hpc_launch = False
         self.bucket_name = InArg('')
         self.folder_path = InArg('')
-        self.H5_format = InArg(False)
+        self.format = InArg('h5')
```
Comment (liadomide) [inline xai_components/xai_storage/store_results.py:127]: To maintain the same behaviour as before I think we should have npz here

**Review by liadomide — COMMENTED** [review]
The `cohort` new param in Siibra component will come as a separate PR, or should we wait with merging this until that is also done ?

**Review by liadomide — APPROVED** [review]

**Comment by romina1601** [comment]
I think the user might be confused if, after he selects a value from the dropdown, he sees a new "Literal String". I think calling it a "Literal Enum" or "Storage format enum" or something like that would be more fitted.
Also, since a Literal String is created, you can then double click on it and edit it as you like... I don't think that this should be allowed. 
But maybe you have another opinion?

**Review by liadomide — APPROVED** [review]

**Comment by liadomide** [comment]
> I think the user might be confused if, after he selects a value from the dropdown, he sees a new "Literal String". I think calling it a "Literal Enum" or "Storage format enum" or something like that would be more fitted. Also, since a Literal String is created, you can then double click on it and edit it as you like... I don't think that this should be allowed. But maybe you have another opinion?

All the above are valid points, but considering the implementation payload, we think the current status is a good-enough version , and an improvement from the previous state.

**Pull request merged**